addons/mail/models/mail_activity.py

```diff
@@ -105,6 +106,7 @@ def default_get(self, fields):
         'Next activities available',
         compute='_compute_has_recommended_activities',
         help='Technical field for UX purpose')
+    active = fields.Boolean('Open', help='Reminder is close (archived) or open', default=True)
```

Comment:
I would rewrite a bit the help, something like "Archive activities that should not be displayed. This is the standard behavior when setting reminders as done instead of unlinking and posting a message."